src/common/config.h

```diff
@@ -361,7 +361,7 @@ const T md_config_t::get_val(const ConfigValues& values,
   return std::get<T>(this->get_val_generic(values, key));
 }
 
-inline std::ostream& operator<<(std::ostream& o, const boost::blank& ) {
+inline std::ostream& operator<<(std::ostream& o, const std::monostate& ) {
```

Comment:
nit: not know but how about killing the extra space before `)`?

Reply:
removed.